v2/event/event_unmarshal.go

```diff
@@ -88,10 +85,10 @@ func ReadJson(out *Event, reader io.Reader) error {
 			// Now we have a specversion, so drain the token queue
 			switch eventContext := out.Context.(type) {
 			case *EventContextV03:
-				for i := 0; i < tokenQueue.i; i++ {
-					val := tokenQueue.slice[i].value
+				for _, entry := range tokenQueue {
```

Comment:
ranging is faster than iterating on indexes? Can you bench with indexes?

Reply:
I did, no diff.